quicktest.go

```diff
@@ -105,14 +105,27 @@ func (c *C) Assert(got interface{}, checker Checker, args ...interface{}) bool {
 // A panic is raised when Run is called and the embedded concrete type does not
 // implement Run, for instance if TB's concrete type is a benchmark.
 func (c *C) Run(name string, f func(c *C)) bool {
-	if r, ok := c.TB.(runner); ok {
-		return r.Run(name, func(t *testing.T) {
-			c := New(t)
-			defer c.Cleanup()
-			f(c)
-		})
+	r, ok := c.TB.(interface {
+		Run(string, func(*testing.T)) bool
+	})
+	if !ok {
+		panic(fmt.Sprintf("cannot execute Run with underlying concrete type %T", c.TB))
 	}
-	panic(fmt.Sprintf("cannot execute Run with underlying concrete type %T", c.TB))
+	return r.Run(name, func(t *testing.T) {
+		c := New(t)
+		defer c.Cleanup()
+		f(c)
+	})
+}
+
+func (c *C) Parallel() {
+	r, ok := c.TB.(interface {
```

Comment:
s/r/p/ ? I guess r was for runner before.

Reply:
done